src/components/Home/CarouselSlider/SubcarouselComponents/SlideContent.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const SlideContent = ({activeIndex, imageSlider}) => {
+	return (
+		<section>
+			{imageSlider &&
+				imageSlider.map((slide, index) => (
+					<div
+						key={index}
+						className={index === activeIndex ? "slides active" : "inactive"}>
+						<img className='slide-image' src={slide.urls} alt='' />
```

Comment:
Don't keep the alt tag empty.

Reply:
I'm added text for alt attribute in the img tag. :)